frameit/lib/frameit/editor.rb

```diff
@@ -347,10 +350,64 @@ def build_title_images(max_width, max_height)
           i.interline_spacing interline_spacing if interline_spacing
           i.fill fetch_config[key.to_s]['color']
         end
-        title_image.trim # remove white space
+
+        # Trimming the image will result in the loss of the common baseline between the text in all images.
+        # title_image.trim # remove white space
```

Comment:
Should we just remove these lines instead of leaving it commented?

Reply:
I left it in for the code review as it was the root cause for the issue.
I agree that the commented `.trim` can be removed.
I suggest to merge the comment "Trimming the image will result..." in the comment a few lines below that starts with "Hence retrieve the calculated...", since it's the rational for the crop solution.

Reply:
Maybe make the comment "Naively trimming the image with `.trim` will..."?

Reply:
That's cool @funnel20, you can make that whatever you'd like, just wanted to make sure you thought it was necessary.

Reply:
@janpio 👍

Reply:
# Natively trimming the image with .trim will result in the loss of the common baseline between the text in all images. Hence retrieve the calculated trim bounding box without actually trimming. See: http://www.imagemagick.org/script/escape.php

Reply:
thats great @funnel20 !

Reply:
I actually really meant "Naively", as in "using the most obvious function that comes to mind to do the job of trimming". But yours probably works as well ;)

Reply:
@janpio I see :-) 
It's a thinker though, as one might suspect a typo (like I did)